packages/apps/spaces/ui/overlay/AlertDialog/AlertDialog.tsx

```diff
@@ -39,7 +45,7 @@ export const AlertDialogOverlay = forwardRef<
     <RadixAlertDialog.Overlay
       ref={ref}
       className={twMerge(
-        ' backdrop-brightness-[.55] bg-blackA6 data-[state=open]:animate-overlayShow fixed inset-0 backdrop-brightness-50"',
+        'z-[5000] backdrop-brightness-[.55] data-[state=open]:animate-overlayShow fixed inset-0',
```

Comment:
Add a comment to recheck this after we remove chakra. I don't think zindex is needed at all once chakra portals go away.